common/common.go

```diff
@@ -22,3 +22,10 @@ const (
 	// DisableDefaultInstallEnvVar is an env variable to disable the default instance
 	DisableDefaultInstallEnvVar = "DISABLE_DEFAULT_ARGOCD_INSTANCE"
 )
+
+// InfraNodeSelector is openshift label for infrastructure nodes since maps are not constants returning from a funciton
```

Comment:
You dont have to explain the technical reasons for using a function in the comments. If you still think it is useful .. Please correct the typo "funciton" to "function".

Reply:
Sure Abhishek, will remove this comment